pysr/__init__.py

```diff
@@ -8,6 +8,7 @@
 from .export_jax import sympy2jax
 from .export_torch import sympy2torch
 from .sr import PySRRegressor
+from .ssr import PySRSequenceRegressor
```

Comment:
Can you give this a more descriptive filename? The `sr.py` is stuck with that name for historical reasons, but I would have preferred to call it `regressor.py`. So you could call the sequence one `regressor_sequence.py`?